docs/installation/linux.md

```diff
@@ -2,7 +2,7 @@ Package installation instructions
 =================================
 
 Supports [Ubuntu 14.04][u14], [Ubuntu 16.04][u16],
-[CentOS 7][cos], [many Linux distributions (AppImage)][lai], and [macOS 10.11][mac].
+[CentOS 7][cos], [Arch Linux][arch], [many Linux distributions (AppImage)][lai], and [macOS 10.11][mac].
 All packages are available on our GitHub [releases][] page.
 
 All of these steps can be done automatically by the [`download.sh`][download] script.
```

Comment:
Today, I'd error on @andschwa's side, give as much information as possible.

In the future, we're going to have to figure out what "support" means, but that's in a post-alpha/beta world.